qutebrowser/browser/qutescheme.py

```diff
@@ -201,6 +203,26 @@ def qute_bookmarks(_url):
     return 'text/html', html
 
 
+@add_handler('tabs')
+def qute_tabs(_url):
+    """Handler for qute://tabs. Display information about all open tabs."""
+    tabs = collections.defaultdict(list)
+    for win_id, window in objreg.window_registry.items():
+        if sip.isdeleted(window):
+            continue
+        tabbed_browser = objreg.get('tabbed-browser',
+                                    scope='window',
+                                    window=win_id)
+        for tab in tabbed_browser.widgets():
+            if not tab.url().url().startswith("qute://tabs"):
```

Comment:
Instead of treating the URL as a string, you should be able to do `if tab.url() != QUrl("qute://tabs"):`.